src/kontrol/prove.py

```diff
@@ -71,6 +74,22 @@ def foundry_prove(
         else []
     )
 
+    if prove_options.cse:
+        test_suite = collect_tests(foundry, tests, reinit=prove_options.reinit, return_empty=True)
+        for test in test_suite:
+            if not isinstance(test.method, Contract.Method) or test.method.function_calls is None:
+                continue
+
+            test_version_tuples = [
+                parse_test_version_tuple(t) for t in test.method.function_calls if t not in summary_ids
+            ]
+
+            if len(test_version_tuples) > 0:
+                print(f'For test {test.name}, found external calls: {test_version_tuples}')
```

Comment:
We should be logging this, not printing it. Printing is for outputting information we expect the next tool to consume.

```suggestion
                _LOGGER.info(f'For test {test.name}, found external calls: {test_version_tuples}')
```